### DISCUSSION TITLE
Consistent tags for UI elements

### DISCUSSION THREAD

**Discussion by sed-i** [grafana/grafana#45368] [General]
The user and password inputs have a name tag:
```html
<input class="css-mtvhwr-input-input" name="user" ...>
<input class="css-mtvhwr-input-input" id="current-password" name="password" ...>
```
But the log in button does not:
```html
<button class="css-w9m50q-button" aria-label="Login button"><span class="css-1mhnkuh">Log in</span></button>
```
So for testing I am currently using xpath:
```python
d.find_element(By.XPATH, "//*[text()='Log in']")
```

It would be handy if key elements in the UI would have more consistent tags.

I am currently using v8.1.1 (90c87a5).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
